gbasis/density.py

```diff
@@ -0,0 +1,44 @@
+"""Density Evaluation."""
+import numpy as np
+
+
+def eval_density(charge_density, cart_cont_orb):
+    """Return the evaluation of the density at different points.
+
+    Parameters
+    ----------
+    charge_density : np.ndarray(L,L,)
+        Charge density matrix.
+    cart_cont_orb : np.ndarray(L,N,)
+        Array of cartesian contracted orbitals at different grid points.
```

Comment:
I think it would be better to say that these are evaluations of a set of orbitals at different grid points. No need to reference cartesian contracted.

Could you change the name to something like `orb_eval`?

Reply:
done.